src/middleware/jira-symmetric-jwt-middleware.ts

```diff
@@ -8,8 +8,8 @@ import { matchRouteWithPattern } from "~/src/util/match-route-with-pattern";
 import { fetchAndSaveUserJiraAdminStatus } from "middleware/jira-admin-permission-middleware";
 
 export const jiraSymmetricJwtMiddleware = async (req: Request, res: Response, next: NextFunction) => {
-
-	const token = req.query?.["jwt"] || req.cookies?.["jwt"] || req.body?.["jwt"];
+	const authHeader = req.headers["authorization"];
+	const token = req.query?.["jwt"] || req.cookies?.["jwt"] || req.body?.["jwt"] || authHeader?.substring(4);
```

Comment:
The Incredibles have told us that they'll be using the auth header to pass the JWT. As mentioned in the description for this PR, stuff on their end won't be ready until late June so we can't test the full flow but I have tested this locally from a browser and all looks good.